src/main/java/org/gridsuite/network/map/dto/mapper/BusBarSectionInfosMapper.java

```diff
@@ -41,4 +44,13 @@ public static BusBarSectionFormInfos toFormInfos(Identifiable<?> identifiable) {
         return builder.build();
     }
 
+    public static BusBarSectionListInfos toListInfos(Identifiable<?> identifiable) {
+        BusbarSection busBarSection = (BusbarSection) identifiable;
+        return BusBarSectionListInfos.builder()
+                .id(busBarSection.getId())
+                .name(busBarSection.getOptionalName().orElse(null))
+                .operatingStatus(toOperatingStatus(busBarSection))
```

Comment:
we're missing
`                .type(identifiable.getType())`
here, no ? 
it's in ElementInfosMapper?